src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs

```diff
@@ -114,6 +114,112 @@ class Program2
 </Workspace>");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddFileBanner)]
+        [WorkItem(33251, "https://github.com/dotnet/roslyn/issues/33251")]
+        public async Task TestSingleLineDocCommentBanner()
+        {
+            await TestInRegularAndScriptAsync(
+@"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <Document>[||]using System;
+
+class Program1
+{
+    static void Main()
+    {
+    }
+}
+        </Document>
+        <Document>/// This is the banner
+/// It goes over multiple lines
+
+class Program2
+{
+}
+        </Document>
+    </Project>
+</Workspace>",
+@"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <Document>/// This is the banner
+/// It goes over multiple lines
```

Comment:
doesn't this produce a warning about a doc comment on an invalid element... i'm not sure we should treat this as a file banner.